DecoderError/explainer.md

```diff
@@ -0,0 +1,107 @@
+# Decoder Error
+
+**Authors**
+* [Nishitha Burman Dey](https://github.com/nishitha-burman)
+Much of this explainer synthesizes and consolidates prior discussions and contributions from [Diego Perez Botero](https://github.com/Diego-Perez-Botero), [Henrik Boström](https://github.com/henbos), [Philipp Hancke](https://github.com/fippo), [Sun Shin](https://github.com/xingri), and other members of the WebRTC working group.
```

Comment:
nit: You should add an extra blank line here so that the rendered markdown document looks right